react/features/youtube-player/index.js

```diff
@@ -0,0 +1,8 @@
+export { default as VideoShareButton } from './components/VideoShareButton';
```

Comment:
This should've been taken care of by the `export * from './components';` below, if not there is a bug somewhere there.